src/sdk-reference/java/1/essentials/error-handling/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: sdk.html.hbs
+algolia: true
+title: Error Handling
+description: How to handle errors with the SDK
+order: 100
+---
+
+# Error Handling
+
+Each of the SDK methods is likely to fail and throw an exception.  
```

Comment:
Similar to my previous remarks about the C++ documentation, this description needs to be changed.

Also, this is not mentioned, but I do hope that our Java `KuzzleException` class inherits the standard `Throwable` class.

Proposal:

```
SDK methods handle failure by throwing exceptions inheriting the `KuzzleException` class, which is a standard Java exception extending the `Throwable` class.
```

Everything else can be removed between that proposal and the sentence starting with "You can find a detailed list..."